content/en/docs/concepts/configuration/even-pods-spread.md

```diff
@@ -0,0 +1,268 @@
+- Change `whenUnsatisfiable: DoNotSchedule` to `whenUnsatisfiable: ScheduleAnyway` to ensure the incoming pod to be always schedulable (suppose other scheduling APIs are satisfied). However, it’s preferred to be placed onto the topology domain which has fewer matching pods. (Be aware that this preferability is jointly normalized with other internal scheduling priorities like resource usage ratio, etc.)
+
+### Example: Multiple TopologySpreadConstraints
+
+Let’s reuse the above example: suppose we have a 4-node cluster where 3 pods labeled `foo:bar` are located from node1 to node3 respectively (`P` represents Pod):
+
+```
++---------------+---------------+
+|     zoneA     |     zoneB     |
++-------+-------+-------+-------+
+| node1 | node2 | node3 | node4 |
++-------+-------+-------+-------+
+|   P   |   P   |   P   |       |
++-------+-------+-------+-------+
+```
+
+We can use 2 TopologySpreadConstraints to control the pods spreading on both zone and node:
+
+```
+kind: Pod
+apiVersion: v1
+metadata:
+  name: mypod
+  labels:
+    foo: bar
+spec:
+  topologySpreadConstraints:
+  - maxSkew: 1
+    topologyKey: zone
+    whenUnsatisfiable: DoNotSchedule
+    labelSelector:
+      matchLabels:
+        foo: bar
+  - maxSkew: 1
+    topologyKey: node
+    whenUnsatisfiable: DoNotSchedule
+    labelSelector:
+      matchLabels:
+        foo: bar
+...
+```
+
+In this case, to match the first constraint, the incoming pod can only be placed onto "zoneB"; while in terms of the second constraint, the incoming pod can only be placed onto "node4". Then the results of 2 constraints are ANDed, so we get the only option "node4".
+
+Be aware that multiple constraints can sometimes lead to conflicts. Suppose we have a 3-node cluster across 2 zones:
+
+```
++---------------+-------+
+|     zoneA     | zoneB |
++-------+-------+-------+
+| node1 | node2 |  nod3 |
++-------+-------+-------+
+|  P P  |   P   |  P P  |
++-------+-------+-------+
+```
+
+If we apply "mypod.yaml" to this cluster, you will notice "mypod" stays in `Pending` state. This is because: to satisfy the first constraint, "mypod" can only be put to "zoneB"; while in terms of the second constraint, "mypod" can only put to "node2". Then a joint result of "zoneB" and "node2" returns nothing.
+
+To overcome this situation, you can either increase the `maxSkew` or modify one of the constraints to use `whenUnsatisfiable: ScheduleAnyway`.
+
+### Conventions
+
+There are some implicit conventions worth noting here:
+
+- Only the pods holding the same namespace as the incoming pod can be matching candidates.
+
+- Nodes without `topologySpreadConstraints[*].topologyKey` present will be bypassed - i.e. the pods located on those nodes being disregarded. In the above example, suppose a node5 carrying label `{zone-typo: zoneC}` joins the cluster, it will be bypassed due to the absence of label key `zone`.
```

Comment:
s/i.e. the pods located on those nodes being disregarded/i.e., pods scheduled on those nodes do not impact maxSkew calculation

Reply:
when you say bypassed, does this also mean that the node will be filtered (i.e., the incoming pod will not be scheduled on it), if yes we should clarify that as well?

Reply:
You're right, it means both (1) pods on those nodes won't impact maxSkew calculation, and (2) the incoming pod won't be scheduled onto this kind of nodes.